### ISSUE TITLE
rtpengine after rtpengine_delete() receive reversed to-tag and from-tag

### ISSUE THREAD

**Issue by kertor** [OpenSIPS/opensips#1339]
Hello collegues! Please, help me to understand the problem.
I call to one sip-account which is registered on two UA - SIPP and CSipSimple.
Order:
1) Two rtpengine_offer(via-branch=extra) - 
first branch, CSipSimple, opensips transmits to rtpengine:
```
"via-branch": "z9hG4bK-524287-1---74592b52732b1722.1"
"from-tag": "d860c85b"
```
second branch, SIPP, opensips transmits to rtpengine:
```
"via-branch": "z9hG4bK-524287-1---74592b52732b1722.2"
"from-tag": "d860c85b"
```
2) Two rtpengine_answer(via-branch=extra) -
second branch(SIPP) 183 reply, opensips transmits to rtpengine:
```
"via-branch": "z9hG4bK-524287-1---74592b52732b1722.2"
"from-tag": "d860c85b"
"to-tag": "29550SIPpTag012"
```
first branch(CSipSimple) 200 reply, opensips transmits to rtpengine:
```
"via-branch": "z9hG4bK-524287-1---74592b52732b1722.1"
"from-tag": "d860c85b"
"to-tag": "ri0a3V2qRHaad.go0ffMP3aOPKnk.3UZ"
```
3) After receives 200ok from CSipSimple, opensips send CANCEL to SIPP. Opensips received "487 Request Terminated" from SIPP and call rtpengine_delete(via-branch=extra), params:
```
"via-branch": "z9hG4bK-524287-1---74592b52732b1722.2"
"from-tag": "29550SIPpTag012"
"to-tag": "d860c85b"
```
Audibility lost after 30 secs after delete branch of failed call.
In logs for 487 reply i see: $tt=29550SIPpTag012 $ft=d860c85b
As I understand, this is not exactly the right behavior - https://github.com/sipwise/rtpengine/issues/518#issuecomment-379739837
It`s correct behavior? Thanks!

Previous ticket: https://github.com/OpenSIPS/opensips/issues/1327

**Comment by razvancrainea** [comment]
It looks like this was an old bug in the module, that was mixing the order of the from and to tags. I have just committed a fix in the master branch, but will also fix the 2.4 and 2.3 and 2.2 branches. Please test again and let us know how it goes.

**Comment by kertor** [comment]
Thank you! I will check the fix in the near future and provide a report.
